buttons.css

```diff
@@ -0,0 +1,80 @@
+@import url(colors.css);
+
+.button-container{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 2rem;
+    align-items: center;
+}
+
+button{
```

Comment:
You can use `btn` class to give these properties. In the future, if you don't want these properties in any button then simply remove `btn` class from the button.

Reply:
Hi tanishq, using button element selector serves the same purpose.